google-http-client/src/main/java/com/google/api/client/util/DateTime.java

```diff
@@ -269,6 +277,98 @@ public int hashCode() {
    *     time zone shift but no time.
    */
   public static DateTime parseRfc3339(String str) throws NumberFormatException {
+    return parseRfc3339WithNanoSeconds(str).toDateTime();
+  }
+
+  /**
+   * Parses an RFC3339 timestamp to a pair of seconds and nanoseconds since Unix Epoch.
+   *
+   * @param str Date/time string in RFC3339 format
+   * @throws NumberFormatException if {@code str} doesn't match the RFC3339 standard format; an
```

Comment:
This does not seem like the right exception type. IllegalArgumentException would be better.

Reply:
I've changed the declared exception type.